dev/release/check-rat-report.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
```

Comment:
I found this by looking at the license check on main:
https://github.com/apache/arrow-rs-object-store/actions/runs/13977939356/job/39136223689

![Screenshot 2025-03-20 at 3 44 46 PM](https://github.com/user-attachments/assets/6383276d-78a3-46dc-8770-5262bad93111)